sklearn/linear_model/coordinate_descent.py

```diff
@@ -614,6 +614,24 @@ class ElasticNet(LinearModel, RegressorMixin):
         number of iterations run by the coordinate descent solver to reach
         the specified tolerance.
 
+    Examples
+    --------
+    >>> from sklearn.linear_model import ElasticNet
+    >>> from sklearn.datasets import make_regression
+    >>>
+    >>> X, y = make_regression(n_features=2, random_state=0)
+    >>> regr = ElasticNet(random_state=0)
+    >>> regr.fit(X, y)
+    ElasticNet(alpha=1.0, copy_X=True, fit_intercept=True, l1_ratio=0.5,
+          max_iter=1000, normalize=False, positive=False, precompute=False,
+          random_state=0, selection='cyclic', tol=0.0001, warm_start=False)
+    >>> regr.coef_
+    array([ 18.83816048,  64.55968825])
+    >>> regr.intercept_
+    1.4512607561654027
+    >>> regr.predict([[0,0]])
```

Comment:
It's strange. I did pep8 on the file before committing. Also, there is the same error in model lasso example. I will fix it.